command/command.go

```diff
@@ -23,9 +24,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+// PrintAppDetails prints some information about the app
+func PrintAppDetails(app *kingpin.Application) {
+	fmt.Printf("help: %s\n", app.Help)
```

Comment:
Use fmt.Println instead of fmt.Printf to simplify this line as it does not require any formatting.